build/IPDK_Container/scripts/set_hugepages.sh

```diff
@@ -1,16 +1,27 @@
+#!/usr/bin/bash
 #Copyright (C) 2021 Intel Corporation
 #SPDX-License-Identifier: Apache-2.0
 
-#!/usr/bin/bash
-
 #...Setting Hugepages...#
-mkdir /mnt/huge
+mkdir -p /mnt/huge
 mount -t hugetlbfs nodev /mnt/huge
 echo -e "nodev /mnt/huge hugetlbfs\n" >> /etc/fstab
 
 echo "vm.nr_hugepages = 4096" >> /etc/sysctl.conf
 #sysctl -p /etc/sysctl.conf
 
-echo 1024 | sudo tee /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages
-echo 1024 | sudo tee /sys/devices/system/node/node1/hugepages/hugepages-2048kB/nr_hugepages
+#
+# Check if the node version of hugepages exists, and set hugepages if so.
+#
+if [ -d /sys/devices/system/node/node0/hugepages-2048kB ] ; then
```

Comment:
Need to correct the folder path. Correct path is 
/sys/devices/system/node/node0/hugepages/hugepages-2048kB

Also requires additional change for checking NUMA node1 existence too before running the configuration settings.
 /sys/devices/system/node/node0/hugepages/hugepages-2048kB
---
 /sys/devices/system/node/node1/hugepages/hugepages-2048kB

Reply:
Somehow some part of the comment text using different formatting options.

Reply:
Corrected the path.